common/lib/task.py

```diff
@@ -88,17 +88,22 @@ def status_error(self, err):
             "message": err
         })
 
-    def upload_file(self):
+    def upload_results(self):
         """upload file to S3"""
-        filename = self.results["artifact"]
-        base = os.path.basename(filename)
-        self.key = f'_tasks/artifacts/{self.task_id}/{base}'
-
-        self.s3_client.upload_file(
-            Filename=filename,
-            Bucket=self.bucket,
-            Key=self.key,
-        )
+        results_dir = self.results["artifact"]
+        self.key = f'_tasks/artifacts/{self.task_id}/'
+
+        for filename in os.listdir(results_dir):
+            print(f'filename: {filename}')
```

Comment:
Not sure if this is just for debugging but if not, can you use `self.logger` for these? Then we can control/standardize log levels a bit easier

Reply:
This is cruft I'll clean up.